index.js

```diff
@@ -210,10 +210,13 @@ var lastResorts = [
   ['BEER', 'Beerware'],
   ['BOOST', 'BSL-1.0'],
   ['BSD', 'BSD-2-Clause'],
+  ['CDDL', 'CDDL-1.1'],
```

Comment:
Hi @aviadatsnyk. What about version 1.0? Is it safe to assume that 1.1 is the version people want?
What is the motivation for this change?
Thanks